changelogs/fragments/56126-eos-without-become.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- eos: don't fail modules without become set, instead show message and continue
```

Comment:
Colons in changelog fragments trip me up too.  You can use a dash or you can quote the line to get rid of the yaml error.